### DISCUSSION TITLE
[@types/oracledb] type error when using type ExecuteOptions after upgrade to type definitions version 6.2.1

### DISCUSSION THREAD

**Discussion by doberkofler** [DefinitelyTyped/DefinitelyTyped#68758] [Issues with a @types package]
When using the latest TypeScript type definitions version 6.2.1.

It seems as if the typeScript type definition for `ExecuteOptions` is wrong as it specifies `type` as number instead of the new `DbType`:

```typescript
        fetchInfo?:
            | Record<
                string,
                {
                    type: number;
                }
            >
            | undefined;
```

When for example using:

```typescript
const options: oracledb.ExecuteOptions = {
		fetchInfo: {
			TEXT: {type: oracledb.STRING},
		},
	};
```
TypeScript complains that `oracledb.STRING` is not of type `number`

[a comment by an automated account is not shown]

**Comment by dannyb648** [comment]
Hi @doberkofler, 

Yeah the 6.2.1 types are definitely not quite right. As you can see from #68727 the types have been merged in before we could test them (and stop this). We're currently trying to fix this line of code on the 6.3.x line which is being tested at the moment. I'm really sorry about this, you are welcome to make a patch for 6.2.2 which undoes this whilst we fix 6.3.x

**Comment by dannyb648** [comment]
In fact @doberkofler, I would suggest reverting to 6.2.0 for now.
